modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro

```diff
@@ -452,6 +452,12 @@ definition {
 			value1 = "${nodeLabel}");
 	}
 
+	macro viewToolTipButtonSourceDiagram {
```

Comment:
The macro name isn't clear, can be more comprehensive

Reply:
Done